PKIndex/PKIndex/PKIndexList.m

```diff
@@ -0,0 +1,167 @@
+//
+//  PKIndexList.m
+//  PKIndex
+//
+//  Created by Fatima Zenine Villanueva on 7/27/15.
+//  Copyright (c) 2015 apps. All rights reserved.
+//
+
+#import "PKIndexList.h"
+
+@implementation PKIndexList
++ (NSArray *)allPokemon {
```

Comment:
This is cool!

To take things further you could create a Pokemon class, that includes properties such as name, type, filename, etc.  Your PKIndexList could then contain an array of Pokemon.  This array could be populated by using the API.

You could then create methods that get Pokemon objects alphabetically, or by type.   Your controllers can then access these methods to get the Pokemon it needs.

This will separate concerns and create a much cleaner controller.